config/initializers/apipie.rb

```diff
@@ -15,6 +15,7 @@
   config.default_locale = FastGettext.default_locale
   config.locale = ->(loc) { loc ? FastGettext.set_locale(loc) : FastGettext.locale }
 
+  ActiveRecord::Migration.maintain_test_schema!
```

Comment:
If it helps, you can simply wrap the lines below in a lambda like the `:template_kinds` and they'll get lazily evaluated.